### PR TITLE
Add @manics @joshmoore to maintainers

### PR DESCRIPTION
**Pull request by manics**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* ~[ ] Reset the build number to `0` (if the version changed)~
* ~[ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)~
* ~[ ] Ensured the license file is being packaged.~

Adding myself as a maintainer in https://github.com/conda-forge/zeroc-ice-feedstock/pull/6 didn't work, so it presumably needs to be done on this main branch.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -57,3 +57,4 @@ about:
 extra:
   recipe-maintainers:
     - dpryan79
+    - manics
```
Comment (manics) [inline recipe/meta.yaml:61]:
```suggestion
    - joshmoore
    - manics
```

**Comment by manics** [comment]
@dpryan79 would you mind merging this? Thanks!

**Comment by dpryan79** [comment]
@manics Done!

**Comment by manics** [comment]
Thanks!

**Pull request merged**